apps/web/src/constants/routes.ts

```diff
@@ -72,6 +72,23 @@ export enum ROUTES {
   STUDIO_ONBOARDING_SUCCESS = '/studio/onboarding/success',
 
   LOCAL_STUDIO_AUTH = '/local-studio/auth',
+
+  /** Account settings routes, bookmarkable by tabs */
+  MANAGE_ACCOUNT_SEGMENT = '/manage-account/',
+  MANAGE_ACCOUNT_USER_PROFILE = '/manage-account/user-profile',
+  MANAGE_ACCOUNT_ACCESS_SECURITY = '/manage-account/access-security',
+  MANAGE_ACCOUNT_ORGANIZATION = '/manage-account/organization',
+  MANAGE_ACCOUNT_TEAM_MEMBERS = '/manage-account/team-members',
+  MANAGE_ACCOUNT_BILLING = '/manage-account/billing',
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Is there supposed to b  a branding route here? I see we have it as a route segment below?‏